src/main/java/ch/njol/skript/classes/data/DefaultFunctions.java

```diff
@@ -711,6 +705,246 @@ public String[] executeSimple(Object[][] params) {
 						"\t\tset {_money} to formatNumber({money::%sender's uuid%})",
 						"\t\tsend \"Your balance: %{_money}%\" to sender")
 			.since("2.10");
+
+		Functions.registerFunction(new SimpleJavaFunction<Number>("mean", new Parameter[]{
+			new Parameter<>("numbers", DefaultClasses.NUMBER, false, null)
+		}, DefaultClasses.NUMBER, true) {
+			@Override
+			public Number @Nullable [] executeSimple(Object[][] params) {
+				Double total = 0d;
+				int length = params[0].length;
+				for (int i = 0; i < length; i++) {
+					Number number = (Number) params[0][i];
+					if (Double.isInfinite(number.doubleValue()) || Double.isNaN(number.doubleValue()))
+						return null;
+					if (total.isInfinite() || total.isNaN())
+						return null;
+					total += (number.doubleValue() - total) / (i + 1);
```

Comment:
is there any reason you chose this technique instead of `+= number.doubleValue() / length`?